### PR TITLE
Working on memory issue during tests for TorchSharp

### PR DESCRIPTION
**Pull request by michaelgsharp**
We are excited to review your PR.

So we can do the best job, please check:

- [ ] There's a descriptive title that will make sense to other developers some time from now. 
- [ ] There's associated issues. All PR's should have issue(s) associated - unless a trivial self-evident change such as fixing a typo. You can use the format `Fixes #nnnn` in your description to cause GitHub to automatically close the issue(s) when your PR is merged.
- [ ] Your change description explains what the change does, why you chose your approach, and anything else that reviewers should know.
- [ ] You have included any necessary tests in the same PR.

### CODE REVIEW

**Comment by ericstj** [comment]
Seems like `CA2213` should have caught these, but it's disabled.  It would be good to re-enable it and ensure it catches these problems and ensure we got them all.  Separately we might need to have an audit of all of our ruleset (compare with runtime) to see if we have others disabled.

**Comment by michaelgsharp** [comment]
So I just tried to enable CA2213 and we have a LOT of issues with it in our repo. I think it would be better to have a separate PR that does just that by itself, as it seems like it will be a decent amount of changes.

File: test/Microsoft.ML.TorchSharp.Tests/QATests.cs [hunk 1]
```diff
@@ -88,35 +88,37 @@ public void TestQALargeFileGpu()
...
-            VBuffer<ReadOnlyMemory<char>> answer = default;
-            ReadOnlyMemory<char> trainingAnswer = default;
-            ReadOnlyMemory<char> context = default;
-            ReadOnlyMemory<char> question = default;
-            VBuffer<float> score = default;
-            int correct = 0;
-            int incorrect = 0;
-
-            while (cursor.MoveNext())
+            using (var cursor = transformedData.GetRowCursor(transformedData.Schema["Answer"], transformedData.Schema["Score"], transformedData.Schema["TrainingAnswer"], transformedData.Schema["Context"], transformedData.Schema["Question"]))
```
Comment (tarekgh) [inline test/Microsoft.ML.TorchSharp.Tests/QATests.cs:91]: >using (var cur [](http://example.com/codeflow?start=12&length=14)

nit:
you can do `using var cursor ....` without `( )` and not using `{` block too.

**Review by tarekgh — APPROVED** [review]
LGTM

**Pull request merged**